src/Adapters/Files/GridStoreAdapter.js

```diff
@@ -20,6 +20,17 @@ export class GridStoreAdapter extends FilesAdapter {
     });
   }
 
+  deleteFile(config, filename) {
+    return config.database.connect().then(() => {
+      let gridStore = new GridStore(config.database.db, filename, 'r');
```

Comment:
`r` stands for reading mode here, I honestly expect it not to work for file deletions.